### PR TITLE
[BOT-127] v2 chat teaser

### PR DESCRIPTION
**Pull request by mib-kamel**
/* Please update CHANGELOG.md. Under [Unreleased] section, describe your changes under 80 characters */

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +36,7 @@
     "botframework-directlinejs": "0.9.15",
     "core-js": "2.4.1",
     "jspeech": "^0.1.1",
+    "live-server": "^1.2.0",
```
Comment (barakyo) [inline package.json]: this should be in `devDependencies` and not in `dependencies`

File: samples/teaser-test/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Carla Teaset test page</title>
+  <link rel="stylesheet" href="../release/iframe.css">
```
Comment (barakyo) [inline samples/teaser-test/index.html]: We'll want to include this into the `release` folder as well for the time being.

File: .gitignore [hunk 3]
```diff
@@ -27,3 +27,5 @@ CognitiveServices.js.map
 *.css
 /test/uploads
 .env
+
+!iframe.css
```
Comment (akingbadeayobami) [inline .gitignore]: @mib-kamel I don't think this actually references `samples/release/iframe.css`

File: samples/release/carla-v2.js [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+            .body
+            .appendChild(chatWidget);
+        }
+      };
+    }
+
+    return {inDocumentReady: inDocumentReady, chatHeaderClick: chatHeaderClick, chatWidgetClick: chatWidgetClick};
+  })();
+
+  var _isSmallScreen = document.documentElement.clientWidth <= 768;
```
Comment (akingbadeayobami) [inline samples/release/carla-v2.js]: @mib-kamel Since `_isSmallScreen` is to be used only inside the function below, Can we move it inside

**Pull request merged**